assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
```

Comment:
We're thinking of components as a combination of elements. They are more complex than elements.